src/quantum/subbenchmarks.jsx

```diff
@@ -38,6 +41,26 @@ const Graph = ({ data, name, url }) => (
         />
       </a>
     </div>
+    <ChartJsWrapper
+      title={name}
```

Comment:
Don't pass a title.
You already have one with PerfherderGraphContainer:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/containers/PerfherderGraphContainer/index.jsx#L35